paddle/fluid/operators/fill_zeros_like_op.cc

```diff
@@ -26,6 +26,12 @@ class FillZerosLikeOp : public framework::OperatorWithKernel {
                    "Input(X) of FillZerosLikeOp should not be null.");
     PADDLE_ENFORCE(ctx->HasOutput("Out"),
                    "Output(Out) of FillZerosLikeOp should not be null.");
+
+    if (ctx->IsRuntime() &&
```

Comment:
InferShape only works for `Tensor`... After giving it a second thought, I feel we'd better add a standalone `FillZerosArrayOp` as in [here](https://github.com/Haichao-Zhang/Paddle/blob/40f96b0ea0111c3ffc528fbe964a991439c48c80/paddle/fluid/operators/fill_zeros_array_op.cc).